src/EvoSC.Common/Clients/MxClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Text.Json;
+using EvoSC.Common.Clients.Dtos;
+using Microsoft.Extensions.Logging;
+
+namespace EvoSC.Common.Clients;
+
+public class MxClient
```

Comment:
I generally think it's a good idea to use the ApiClient you've created, however if we decide to use it, I think we should consider moving it to the Evo org. Since you're the only maintainer, if you decide to quit Evo or you no longer do any development, we will have no way to update the client with new features.